handler/PrettyPageHandler.js

```diff
@@ -27,7 +27,7 @@ var util = require('util');
  * @param sendResponse
  * @class
```

Comment:
Can you include additionalScripts in jsdoc as an optional parameter.

Reply:
Addressed in latest commit. There seem to be a few ways to specify any optional parameter, so if there is another preferred style please let me know.